### DISCUSSION TITLE
Failed to request creation of interactive window from VS Code.

### DISCUSSION THREAD

**Discussion by Vincent0706** [microsoft/vscode-jupyter#9906] [Questions and Answers]
I want to open a interactive window from VS Code on wsl. But it report me "Failed to request creation of interactive window from VS Code."

**Comment by DonJayamanne** [comment]
Looks like a bug somewhere, please could you file an issue here https://github.com/microsoft/vscode-jupyter/issues/new?assignees=octocat&labels=bug%2Ctriage-needed&template=2_bug_form.yml with the output from the logs along with the versions of VS Code and jupyter extension.

**Reply by Vincent0706** [reply]
today, I try again and find it have been ok.....
maybe have some other error at that time.
